content/soarian/dstu2/care-provision/careplan.md

```diff
@@ -46,23 +46,29 @@ _Implementation Notes_
 
 * The API returns the most current careplan list.
 
-* The Narrative includes additional information pertaining to the careplan, and should be shown to the user to ensure completeness of clinical content and context.
+* The API returns values in reverse chronological order by the most currently charted date.
+
+* The API returns charted content that is not marked as erroneous.
```

Comment:
Just state which statuses the API returns specifically, and what statuses won't be returned.